src/lib/milvus/client.py

```
@@ -358,5 +358,19 @@ def search_similar_by_ids(
+    def remove_vectors(self, ids: list[str], store: str, brain_id: str) -> None:
+        """
+        Remove vectors from the vector store.
+        """
+        if not ids:
+            return None
+        client = self._get_client(brain_id)
+        self._ensure_store(store, brain_id)
+        try:
+            client.delete(store, ids=ids)
+        except Exception:
+            pass
```

Ruff (0.14.14)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)